framework/src/modules/nft/internal_method.ts

```diff
@@ -82,20 +106,141 @@ export class InternalMethod extends BaseMethod {
 		});
 	}
 
-	public async transferInternal(
+	public async verifyTransfer(
+		immutableMethodContext: ImmutableMethodContext,
+		senderAddress: Buffer,
+		nftID: Buffer,
+	) {
+		let nft;
+		try {
+			nft = await this._nftMethod.getNFT(immutableMethodContext, nftID);
+		} catch (error) {
+			if (error instanceof NotFoundError) {
+				throw new TransferVerifyError(
+					'NFT does not exist',
+					NftEventResult.RESULT_NFT_DOES_NOT_EXIST,
+				);
+			} else {
```

Comment:
no need to have `else` here. Applies to other places as well

Reply:
all relevant places updated under [this](https://github.com/LiskHQ/lisk-sdk/pull/9005/commits/36c2039f8e2b41b508cde5346a55bf1c2dc1d59d) commit